Assets/Scripts/EnemysClearedTrigger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemysClearedTrigger : MonoBehaviour
+{
+
+    public bool enemiesPresent;
+
+    public void Start()
+    {
+        enemiesPresent = true;
```

Comment:
you could always default enemiesPresent to true inline instead of in start to simplify things